Tests/Controller/HelperControllerTest.php

```diff
@@ -328,6 +329,20 @@ public function testappendFormFieldElementAction($validatorInterface)
 
         $twig = new \Twig_Environment($this->getMock('\Twig_LoaderInterface'));
         $twig->addExtension(new FormExtension($mockRenderer));
+
+        if (Kernel::MAJOR_VERSION >= 3 && Kernel::MINOR_VERSION >= 2) {
```

Comment:
@cedricziel the bundle depends on individual packages, not on the fullstack repo (to give flexibility to users), but this allows to have different versions of each package. Kernel constants give you the version of HttpKernel, not of the Twig bridge